### PR TITLE
Feature/7 post comment on article

### PR DESCRIPTION
**Pull request by Cam-Bloom**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -112,3 +112,29 @@ describe("/api/articles/:article_id", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  describe("POST", () => {
+    test.only('200: Should respond with newly created comment object when passed a "body" and "username"', () => {
```
Comment (aloush-dev) [inline __tests__/app.test.js]: Make sure your description reflects what you're testing 200 should be 201

File: models/news.models.js [hunk 2]
```diff
@@ -36,4 +36,26 @@ function fetchArticleById(id) {
...
+  const sqlQuery = `
+  INSERT INTO comments
+  (body, author, article_id)
+  VALUES
+  ($1, $2, $3)
+  RETURNING *
+  `;
+
+  return db.query(sqlQuery, inputValues).then(({ rows: [comment] }) => {
+    console.log(comment);
```
Comment (aloush-dev) [inline models/news.models.js]: remove console logs

File: __tests__/app.test.js [hunk 3]
```diff
@@ -112,3 +112,29 @@ describe("/api/articles/:article_id", () => {
...
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author", "butter_bridge");
+          expect(comment).toHaveProperty("body", "Cool new comment");
+          expect(comment).toHaveProperty("article_id", id);
+        });
+    });
+  });
+});
```
Comment (aloush-dev) [inline __tests__/app.test.js:200]: More tests need adding here and then the solutions too.
Mainly for error handling.

Think about if the username doesn't exist.
What if the comment body or username are empty?

**Review by aloush-dev — APPROVED** [review]
Please add some error handling, and the tests to go with it.

File: __tests__/app.test.js (hunk 3 above)
Comment (Cam-Bloom) [reply __tests__/app.test.js:200]: Added the tests but all errors were already handled, doesnt exist by the if statement in models and username by foriegn key violation in postgres error handler, am i alright to merge ?

**Pull request merged**